ecosystem/sep-0010.md

```diff
@@ -112,19 +112,19 @@ GET https://auth.example.com/?account=GCIBUCGPOHWMMMFPFTDWBSVHQRT4DIBJ7AD6BZJYDI
 On success the endpoint must return `200 OK` HTTP status code and a JSON object with these fields:
 
 * `transaction`: an XDR-encoded Stellar transaction with the following:
-  * source account set to server's signing account
+  * source account set to the **Home Domain**'s ``SIGNING_KEY``
   * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
-  * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give user time to sign transaction)
+  * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give the **Client Application** time to sign transaction)
   * operations:
-    * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
-      * The value of key is the home domain the client is authenticating with, followed by `auth`. It can be at most 64 characters.
+    * `manage_data(source: authenticating_account, key: '<home domain> auth', value: random_nonce())`
+      * The value of key is the **Home Domain**, followed by `auth`. It can be at most 64 characters.
```

Comment:
💡 I think it is worth adding a dot point here above the key dot point, that states that the source account is the `**Authenticating Account**` so you make use of the participant bold text here.